Classes/Override/CommandMapPostProcessingHook.php

```diff
@@ -20,6 +20,7 @@
 
 /**
  * Class takes care of content translation for elements within containers
+ * @deprecated
```

Comment:
should at least state when it is scheduled to be removed, e.g. `@deprected will be removed in v4`